src/data/index.ts

```diff
@@ -24,12 +24,12 @@ export { loadFromBlob } from "./blob";
 export { saveAsJSON, loadFromJSON } from "./json";
 export { saveToLocalStorage } from "./localStorage";
 
-const BACKEND_GET = "https://json.excalidraw.com/api/v1/";
+const BACKEND_GET = process.env.REACT_APP_BACKEND_V1_GET_URL;
```

Comment:
Wasn't sure whether these have to be defaulted here but I'm happy to make that change.